futures-test/src/io/interleave_pending.rs

```diff
@@ -0,0 +1,161 @@
+use futures_io::{self as io, AsyncBufRead, AsyncRead, AsyncWrite};
+use pin_utils::{unsafe_pinned, unsafe_unpinned};
+use std::{
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+/// I/O wrapper for interleaving [`Poll::Pending`] in calls to read or write.
+///
+/// See the [`interleave_pending`] and [`interleave_pending_write`] methods.
+///
+/// [`interleave_pending`]: super::AsyncReadTestExt::interleave_pending
+/// [`interleave_pending_write`]: super::AsyncWriteTestExt::interleave_pending_write
+#[derive(Debug)]
+pub struct InterleavePending<IO> {
```

Comment:
I'm not sure what the standard is for 2 character acronyms in Rust's version of PascalCase/UpperCamelCase. As far as I know there's no special-case for them so they should follow the standard acronym rule of just treating it as a single word, which would make this `Io`. (If I remember my .NET conventions correctly they had a special case for < 3 character acronyms to be kept uppercase like this).

Reply:
Done.